src/Endpoint.js

```diff
@@ -521,6 +521,18 @@ export default class Endpoint extends EventEmitter {
       NativeModules.PjSipModule.changeOrientation(orientation)
     }
 
+    changeCodecSettings(codecSettings) {
+        return new Promise(function(resolve, reject) {
+        NativeModules.PjSipModule.changeCodecSettings(codecSettings, (successful, data) => {
```

Comment:
I would like errors to be handled here in this promise. Not sure how to do that. @datso